pyrefinebio/script.py

```diff
@@ -0,0 +1,126 @@
+import click
+import json
+
+import pyrefinebio.high_level_functions as hlf 
+
+from pyrefinebio.exceptions import DownloadError
+
+@click.group()
+def cli():
+    """RefineBio CLI
+    """
+    pass
+
+class DictParamType(click.ParamType):
+    name = "dict"
+
+    def convert(self, value, param, ctx):
+        try:
+            return json.loads(value)
+        except:
+            self.fail(
+                "expected valid string in a dict form, "
+                "got: {0}".format(value),
+                param,
+                ctx
+            )
+
+
+class ListParamType(click.ParamType):
+    name = "list"
+
+    def convert(self, value, param, ctx):
+        try:
+            return value.split()
+        except:
+            self.fail(
+                "expected valid string in list form, "
+                "got: {0}".format(value),
+                param,
+                ctx
+            )
+
+
+@cli.command()
+@click.option("--entity", default="", help="entity to print help for")
+def help(entity=None):
+    """
+    Prints out information about pyrefinebio's classes and functions.
+    To get information about a class method, pass in the class name and method name  
+    separated by either a space or a `.`
+
+    Example:
+        $ refinebio help --entity "Sample.search"
```

Comment:
Any chance we can make this an unnamed arg? `refinebio help "Sample.search"` seems easier to type.